src/wallet/wallet.cpp

```diff
@@ -639,9 +639,7 @@ bool CWallet::EncryptWallet(const SecureString& strWalletPassphrase)
         if (IsHDEnabled()) {
             CKey key;
```

Comment:
nit: remove this unused variable in this commit (it was unused before your PR, but since you're here...)